lldb/tools/debugserver/source/MacOSX/MachProcess.mm

```diff
@@ -1739,7 +1739,7 @@ static uint64_t bits(uint64_t value, uint32_t msbit, uint32_t lsbit) {
     ReplyToAllExceptions();
   }
 
-  m_task.ShutDownExcecptionThread();
+  m_task.ShutDownExceptionThread();
```

Comment:
🤔

Reply:
No idea, it's been that way since forever.